src/components/ShellView/ResultGraph.vue

```diff
@@ -451,9 +458,11 @@ export default {
       // Auto layout after drag
       this.g6Graph.on('node:dragstart', (e) => {
         this.g6Graph.layout();
+        this.refreshDraggedNodePosition(e);
       });
 
       this.g6Graph.on('node:drag', (e) => {
+        this.g6Graph.layout();
```

Comment:
Based on this example: https://g6-v3-2.antv.vision/en/examples/net/forceDirected, you should not relayout during the drag. I think it can also cause lagging because it causes relayout to occur too frequenrly. This is not desirable. Please revert it.